### DISCUSSION TITLE
How to apply exported theme to an app?

### DISCUSSION THREAD

**Discussion by marcelo-estevam-nectar** [microsoft/powerapps-tools#729] [Q&A]
Hi,

If I create a theme with _PowerApps Theme Editor.exe_ it exports the theme as a JSON. How can I apply this JSON to an canvas app?

Kind Regards

Marcelo Almeida Estevam

**Comment by Louicha** [comment]
Hi @marcelo-estevam-nectar ,
if you are using the [theme editor from CoE](https://docs.microsoft.com/en-us/power-platform/guidance/coe/theming-components#theme-editor), you can download you theme as .msapp. 
Afterwards, create a new Canvas App and select Open and then select the .msapp template file downloaded from the theme editor/gallery.
[Theming Components - Use the template](https://docs.microsoft.com/en-us/power-platform/guidance/coe/theming-components#use-the-template)
